rewrite-java-core/src/main/resources/META-INF/rewrite/rewrite.yml

```diff
@@ -6,4 +6,12 @@ recipeList:
   - org.openrewrite.java.ChangePackage:
       oldPackageName: com.azure
       newPackageName: io.clientcore
-      recursive: true
\ No newline at end of file
+      recursive: true
+  - org.openrewrite.text.FindAndReplace:
+      find: 'requires com.azure.core'
+      replace: 'requires com.azure.core.v2'
```

Comment:
Is this rule idempotent? If this rule is run twice I am guessing it will again try to replace the string as the `find` matches the second time too.

initial:
`requires com.azure.core`

first execution of this recipe:
`requires com.azure.core.v2`

second execution of this recipe
`requires com.azure.core.v2.v2`